### PR TITLE
serialize Address

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

**Comment by Thykof** [comment]
*marked as blocked by https://github.com/massalabs/as/issues/89*

File: assembly/std/address.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import { Valider, ByteArray } from '@massalabs/as-types';
+import { Valider, Serializable, Args, Result } from '@massalabs/as-types';
 
 /**
  * A Massa's blockchain address.
  *
  */
-export class Address implements Valider {
+export class Address implements Valider, Serializable {
```
Comment (gregLibert) [inline assembly/std/address.ts:7]: Oh shit Valider still exist ? Maybe we should mark it a deprecated.

File: assembly/std/address.ts [hunk 2]
```diff
@@ -29,81 +29,55 @@ export class Address implements Valider {
...
+    }
+    this._value = result.unwrap();
+    return new Result(args.offset);
   }
 
   /**
    * Serialize to byte string.
    */
-  toByteString(): string {
+  toString(): string {
```
Comment (gregLibert) [inline assembly/std/address.ts:63]: is it a big breaking change ?

**Review by gregLibert — APPROVED** [review]

File: assembly/std/address.ts (hunk 2 above)
Comment (Thykof) [reply assembly/std/address.ts:63]: no because only the name of the function changes, not the logic

File: assembly/std/address.ts (hunk 1 above)
Comment (Thykof) [reply assembly/std/address.ts:7]: okay, issue create here: https://github.com/massalabs/as/issues/92

**Review by peterjah — APPROVED** [review]

**Pull request merged**